### PR TITLE
simplify key handling

### PR DESCRIPTION
**Pull request by resmo**
hmm this something speak against doing the key handling like little ponies would do?

### CODE REVIEW

File: cs.py [hunk 1]
```diff
@@ -195,11 +124,7 @@ def _request(self, command, json=True, opcode_name='command', **kwargs):
             kw['data'] = kwargs
         response = getattr(requests, self.method)(self.endpoint, **kw)
         data = response.json()
-        key = '{0}response'.format(command.lower())
-        if key not in data and key in response_key_overrides:
-            key = response_key_overrides[key]
-            assert key in data, key
-
+        key = list(data.keys())[0]
```
Comment (brutasse) [inline cs.py]: Hmm why not… I think I'd prefer `[key] = data.keys()` to be a little more strict.

**Comment by resmo** [comment]
@brutasse yay, it builds :)

**Comment by brutasse** [comment]
Let's :shipit: then

**Comment by resmo** [comment]
@brutasse wanted to make a shy request for aa tag/release in pip. No need to hurry but if you have time...

and btw time: http://www.meetup.com/Ansible-Geneva/events/221210115/ and you may see your stuff in action :+1:

**Comment by brutasse** [comment]
@resmo thanks for the bump, I forgot to do it the other day… 0.6.3 is up on PyPI :)

Unfortunately I won't be available on April 16… I hope other teammates can make it.

**Comment by resmo** [comment]
On 23.03.2015 14:19, Bruno Renié wrote:
> @resmo thanks for the bump, I forgot to do it the other day… 0.6.3 is up on PyPI :)

Cool thx!

> Unfortunately I won't be available on April 16… I hope other teammates can make it.

Too bad you can not make it, see you next time!

**Pull request merged**